Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -353,6 +354,18 @@ protected void onCreate(Bundle savedInstanceState)
 
     if (NativeLibrary.IsGameMetadataValid())
       setTitle(NativeLibrary.GetCurrentTitleDescription());
+
+    OnBackPressedCallback callback = new OnBackPressedCallback(true)
+    {
+      @Override public void handleOnBackPressed()
```

Comment:
We usually put the `@Override` on its own line.

Reply:
It doesn't seem like an interface is provided to implement this. Maybe I'm missing something?

Reply:
Is that possible since the class takes a boolean here?

Reply:
I don't see why not. Like this:

```suggestion
      @Override
      public void handleOnBackPressed()
```

(Not that I've actually tried it!)

Reply:
Oh you meant like that, I misunderstood